openassessment/xblock/studio_mixin.py

```diff
@@ -81,14 +80,11 @@ def studio_view(self, context=None):  # pylint: disable=unused-argument
             self.STUDIO_EDITING_TEMPLATE
         ).render(self.editor_context())
         fragment = Fragment(rendered_template)
-        if settings.DEBUG:
```

Comment:
yes and no.  From what I understand from talking to people, the debug settings weren't really being exercised.  With the new architecture, a dev build is theoretically possible, which would use a different webpack config and different env variables, but it wouldn't function in the way this old code dictated.
The python would either be agnostic to the change or it would have to somehow be listening to an active webpack dev server